### PR TITLE
[18.0][MIG] product_template_multi_link: Migration to 18.0

### PR DESCRIPTION
**Pull request by thienvh332**

### CODE REVIEW

**Review by sebalix — APPROVED** [review]

File: product_template_multi_link/tests/test_product_template_link.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateLink(TransactionCase):
```
Comment (sebalix) [inline product_template_multi_link/tests/test_product_template_link.py]: You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method

File: product_template_multi_link/tests/test_product_template_link_type.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
+# Copyright 2019 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from psycopg2 import IntegrityError
+
+from odoo.tests import TransactionCase
+from odoo.tools import mute_logger
+
+
+class TestProductTemplateLinkType(TransactionCase):
```
Comment (sebalix) [inline product_template_multi_link/tests/test_product_template_link_type.py]: You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method

File: product_template_multi_link/tests/test_product_template_linker.py [hunk 3]
```diff
@@ -0,0 +1,130 @@
+# Copyright 2020 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import TransactionCase
+
+
+class TestProductTemplateLinker(TransactionCase):
```
Comment (sebalix) [inline product_template_multi_link/tests/test_product_template_linker.py]: You can inherit from `BaseCommon` class and remove the `tracking_disable=True` key from context set in `setUpClass` method

**Review by ajaniszewska-dev — APPROVED** [review]
seems ok

File: product_template_multi_link/tests/test_product_template_linker.py (hunk 3 above)
Comment (thienvh332) [reply product_template_multi_link/tests/test_product_template_linker.py]: I updated it. Thanks :pray:

**Review by sebalix — APPROVED** [review]

**Pull request merged**